scss/components/_search-bar.scss

```diff
@@ -26,7 +26,7 @@
 
   @include media($med) {
     .combo__input {
-      width: calc(100% - 4.6rem);
+      width: calc(100% - 16.6rem);
```

Comment:
What does 16.6 stand for? Seems like a random number. It is being set somewhere else?